src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/commands.py

```diff
@@ -27,4 +28,8 @@ def load_command_table(self, _):
 
     with self.command_group('consumption reservations details') as d:
         d.custom_command('list', 'cli_consumption_list_reservations_details', transform=transform_reservation_details_list_output,
-                         exception_handler=consumption_exception_handler, validator=validate_reservations_details, client_factory=reservations_details_mgmt_client_factory)
+                         exception_handler=consumption_exception_handler, validator=None, client_factory=reservations_details_mgmt_client_factory)
+
+    with self.command_group('consumption pricesheet') as p:
+        p.custom_command('show', 'cli_consumption_list_pricesheet_show', transform=transform_pricesheet_show_output,
+                         exception_handler=consumption_exception_handler, validator=None, client_factory=pricesheet_mgmt_client_factory)
```

Comment:
You can remove the `validator` kwarg

Reply:
sure.. i have another PR I'll publish today, i'll make these changes then. Thanks.

Reply:
@williexu  - Regarding str conversion, if removed, json conversion fails when reading decimal or any non-string type.